content/akeneo-event-platform/getting-started.md

```diff
@@ -1,4 +1,157 @@
 # Getting started
 
-::: panel-link <!-- TODO TEXT PANEL LINK -->[Next](/akeneo-event-platform/api-reference.html)
-:::
\ No newline at end of file
+This guide will walk you through the essential steps to set up your environment, create subscriptions, and trigger events.
+
+### Prerequisites
+
+Before proceeding, ensure that you have an active App (it can be a custom App) or an active connection to an Akeneo PIM.
+
+To learn how to create a connection, see the [Authentication Guide](/documentation/authentication.html#client-idsecret-generation). If you're setting up a custom App, follow the steps in [this tutorial](/tutorials/how-to-get-your-app-token.html#) to obtain an App token.
+
+---
+
+### Basic Use Case: Setting Up Event Subscriptions
+
+This guide provides a step-by-step walkthrough to help you set up event subscriptions using the Akeneo Event Platform. By the end of this guide, you will have created a connection, authenticated your App, subscribed to events, and triggered an event from your PIM.
+
+### 1. Retrieve your credentials from your targeted PIM
+
+::: info
+🛠 For every call to the API, you will need `X-PIM-TOKEN` & `X-PIM-CLIENT-ID` .
+
+In this example, we will create a new `connection` in the PIM and use it to generate an `API token`.
+:::
+
+1. **Create a connection in Akeneo PIM:**
+   - Navigate to **Connect** > **Connection settings** > **Create**.
+   - Fill out the form to create the connection.
+   - Note the generated `Client ID`, `Secret`, `Username`, and `Password`.
+
+2. **Set Your Environment Variables:**
+   - Define the Client ID, Secret, Username, Password, and Akeneo host URL as environment variables:
+    ```bash
+   export CLIENT_ID="your-client-id"
+   export CLIENT_SECRET="your-client-secret"
+   export API_USERNAME="your-API-username"
+   export API_PASSWORD="your-API-password"
+   export AKENEO_HOST="http://your-host"
+    ```
+   Replace the placeholders with your actual credentials and host URL.
+
+3. **Encode Your Credentials:**
+   - Encode the Client ID and Secret in base64 format, separated by a colon (:):
+   ```bash
+   export BASE64_ENCODED_CLIENTID_AND_SECRET=$(echo -n "$CLIENT_ID:$CLIENT_SECRET" | base64)
+   ```
+
+4. **Retrieve Your API Token:**
+   - Make the API call to retrieve your `API token` using the environment variables:
+
+    ```bash
+   curl --request POST "$AKENEO_HOST/api/oauth/v1/token" \
+   --header "Content-Type: application/json" \
+   --header "Authorization: Basic $BASE64_ENCODED_CLIENTID_AND_SECRET" \
+   --data-raw '{
+   "grant_type": "password",
+   "username": "'"$API_USERNAME"'",
+   "password": "'"$API_PASSWORD"'"
+   }'
+    ```
+   You can check the official [token generation documentation](/documentation/authentication.html#token-generation) for further information.
```

Comment:
```suggestion
    ```

    After creating the subscriber, store the id from the response in an environment variable:
```bash
export PIM_API_TOKEN="..."  # Replace with the actual token from the response
```
       
   You can check the official [token generation documentation](/documentation/authentication.html#token-generation) for further information.
```